addons/microcontroller/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Microcontroller
```

Comment:
I think you could go for a more descriptive name for this. controller is already a kind of loaded word in software.